src/traceml_ai/instrumentation/hooks/optimizer_hooks.py

```
@@ -99,3 +99,28 @@ def ensure_optimizer_timing_installed() -> None:
+
+    Inverse of ``ensure_optimizer_timing_installed()``; intended for tests and
+    re-initialization. Best-effort and never raises. After this call,
+    ``wrap_optimizer()`` is permitted again and the auto path can re-install.
+    """
+    global _HANDLES
+    if _HANDLES is not None:
+        for handle in _HANDLES:
+            try:
+                handle.remove()
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)